### PR TITLE
A few wording improvements suggestions

### PR DESCRIPTION
**Pull request by nuxper**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
-You can run osmose-backend in a Docker container. The advantage is that
-you do not need to setup and configure Python, Java and PostgreSQL on
-your system. The PostgreSQL and eventually the Osmose Frontend are
-dependencies.
+osmose-backend can be run in a Docker container. This avoids setting
+up and configuring Python, Java and PostgreSQL on your system.
+The osmose-backend docker image requires a running instance of PostgreSQL
+(PostGIS). An instance of osmose-frontend is also required to display the
+issues on a web interface.
```
Comment (frodrigo) [inline docker/README.md:8]: PostgreSQL is not a requirement as it is integrated into the docker-compose setup.

I prefer avoid to encourage to use the osmose-frontend as the installation and the usage is more tedious than the backend.

Comment (nuxper) [reply docker/README.md:8]: OK. My firt reason to suggest a change for this sentence is that "eventually" is a "faux ami" meaning "finalement / au final" and I think you may have meant "éventuellement"
Alternative proposals ;) :
- remove this last sentence completely or
- "Note : A PostgreSQL docker is automatically installed and run by docker-compose and doesn't need be installed manually." (no mention of osmose-frontend)
- "Note : A PostgreSQL docker is automatically installed and run by docker-compose and doesn't need be installed manually. The osmose-frontend docker may also be run but is not mandatory"

Comment (frodrigo) [reply docker/README.md:8]: The last.

File: docker/README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-You can run osmose-backend in a Docker container. The advantage is that
-you do not need to setup and configure Python, Java and PostgreSQL on
-your system. The PostgreSQL and eventually the Osmose Frontend are
-dependencies.
+osmose-backend can be run in a Docker container. This avoids setting
+up and configuring Python, Java and PostgreSQL on your system.
+
+Note : A PostgreSQL docker is automatically installed and run by 
+docker-compose and doesn't need be installed manually. 
+The osmose-frontend docker may also be run but is not mandatory
```
Comment (frodrigo) [inline docker/README.md:9]:
```suggestion
The osmose-frontend docker may also be run but is not mandatory.
```

**Comment by frodrigo** [comment]
Thank you.

**Pull request merged**